### ISSUE TITLE
Compare delta in assertions for DatabasePerformanceTest

### ISSUE THREAD

**Issue by jwillp** [Morebec/YDB#7]
Sometimes the record creation test fails. 
Meaning it takes more than 1.5 seconds to insert 1000 records with the following schema:
```php
 $schema = new TableSchema($tableName, [
            new Column('id', ColumnType::STRING(), true /* indexed */),
            new Column('first_name', ColumnType::STRING()),
            new Column('last_name', ColumnType::STRING()),
            new Column('age', ColumnType::INTEGER()),
            new Column('indexed_column', ColumnType::INTEGER(), true /* indexed */)
        ]);
```
It would be better for performane improvement debuging to change the assertions
to compare agains the delta rather than doing a `assertTrue`
That way in the tests result we would a more verbose output along the lines
of `failed to assert that 1.67, is less than 1.5`

So, in summary, we'd need to change `assertTrue` to `assertLessThan`
